### PR TITLE
Updates to only one site and with other site sql

### PR DESCRIPTION
**Pull request by nottmjr24**
Have added the tester associated to either a single or multiple site sql changes to this branch for approval. Have tested both locally

### CODE REVIEW

File: src/server/resources/queries/user/MOT_TESTER_CLASS_4_WITH_ONLY_ONE_SITE.sql [hunk 1]
```diff
@@ -1,40 +1,122 @@
...
+  UNION
+    SELECT
+      person_id
+    FROM
+      mot_test_current
+    WHERE
+      status_id = 4
+  )
+  LIMIT 100;
+Collapse
```
Comment (ibrahimjassat) [inline src/server/resources/queries/user/MOT_TESTER_CLASS_4_WITH_ONLY_ONE_SITE.sql]: You may want to review this - got bit of a conversation in here

File: src/server/resources/queries/user/MOT_TESTER_CLASS_4_WITH_ONLY_ONE_SITE.sql [hunk 2]
```diff
@@ -1,40 +1,122 @@
...
\ No newline at end of file
+Karl
+
+
+
+
+
+
+
+Message List
```
Comment (ibrahimjassat) [inline src/server/resources/queries/user/MOT_TESTER_CLASS_4_WITH_ONLY_ONE_SITE.sql]: Details of a conversation in the file

File: src/server/resources/queries/user/MOT_TESTER_CLASS_4_WITH_ONLY_ONE_SITE.sql [hunk 3]
```diff
@@ -1,40 +1,122 @@
...
-    and mtc.status_id = 4) -- exclude any testers with active tests
-and not exists (
-    select 1 from site_business_role_map sbrm
-    where p.id = sbrm.person_id
-    group by sbrm.person_id
-    having count(*)>1)
-and p.username is not null -- exclude dodgy test data
-limit 10
\ No newline at end of file
+Karl
```
Comment (karlbeckettbjss) [inline src/server/resources/queries/user/MOT_TESTER_CLASS_4_WITH_ONLY_ONE_SITE.sql]: This file should be removed from the commit, this is still bing tested and reviewed in another branch

**Review by karlbeckettbjss — CHANGES_REQUESTED** [review]

File: src/server/resources/queries/user/MOT_TESTER_CLASS_4_WITH_ONLY_ONE_SITE.sql (hunk 1 above)
Comment (nottmjr24) [reply src/server/resources/queries/user/MOT_TESTER_CLASS_4_WITH_ONLY_ONE_SITE.sql]: Updated thanks..

File: src/server/resources/queries/user/MOT_TESTER_CLASS_4_WITH_ONLY_ONE_SITE.sql (hunk 2 above)
Comment (nottmjr24) [reply src/server/resources/queries/user/MOT_TESTER_CLASS_4_WITH_ONLY_ONE_SITE.sql]: removed.

**Pull request closed without merging**